browser/src/DownloadsPage/DownloadsPage.tsx

```diff
@@ -198,6 +199,11 @@ const DownloadsPage = ({ location }: Props) => {
             label: 'ExAC',
             render: () => <ExacDownloads />,
           },
+          {
+            id: 'research',
+            label: 'Research',
+            render: () => <ResearchDownloads />,
```

Comment:
A React functional component is just a function, so we could change this to be `render: ResearchDownloads`, and that should work. If so, please go ahead and update the rest of this component.